docs/Ingress.md

```diff
@@ -461,7 +466,7 @@ Istio is a popular choice for its Service Mesh capabilities. However, Service Me
 
 It is important to confirm which Nginx Ingress controller you intended to implement, as they have *very* different annotations and configurations. For these examples, we are using the Kubernetes Ingress Nginx (option 1) and the Nginx Ingress Open Source (option 2).
 
-## Configuring Ingress Using Kubernetes Ingress Nginx <a name="Ingress-Nginx"></a>
+## Configuring Ingress Using Kubernetes Ingress Nginx 
 
 For instructions on how to install and configure the NGINX Ingress Controller, see the 
 [NGINX Ingress Controller GitHub repository](https://github.com/nginxinc/kubernetes-ingress/) and the [Installation Guide](https://kubernetes.github.io/ingress-nginx/deploy/).
```

Comment:
This line needs to be modified https://github.com/nginxinc/kubernetes-ingress/ should be replaced with https://github.com/kubernetes/ingress-nginx